### PR TITLE
💄🐛✨ : Activity/Schedule enhancements and date of birth field for children

### PR DESCRIPTION
**Pull request by d-azmanov**
- Button to clear schedule for a child (deletes all activities for that child)
- Made the child ID a generated field instead of a South African ID
- Added current activity to the ngxs so you can refresh while viewing/editing an activity
- Added current child to the ngxs so you can refresh while editing a child
- Added date of birth field to children

### CODE REVIEW

File: libs/edit-child/feature/src/lib/edit-child.component.ts [hunk 1]
```diff
@@ -215,4 +216,37 @@ export class EditChildComponent implements OnInit {
...
+    await this.serv.getChildren(this.store.snapshot().user.id).toPromise().then(
+      async res=>
+      {
+        await res.forEach( (c: Child) => {
+          if(c.id == this.childDetails.id)
+          {
+            this.childDetails = c;
+          }          
+        }); 
+        console.log("the absolit foiund child: ", this.childDetails);
```
Comment (PxttytheThxtty) [inline libs/edit-child/feature/src/lib/edit-child.component.ts:232]: Lol

File: libs/parent-add-activity/feature/src/lib/parent-add-activity.component.ts [hunk 2]
```diff
@@ -48,6 +79,8 @@ export class ParentAddActivityComponent implements OnInit{
   //Populate the activityDetails object from form input
   async getActivityValues(val : any)
   {  
+    console.log(val);
```
Comment (PxttytheThxtty) [inline libs/parent-add-activity/feature/src/lib/parent-add-activity.component.ts:82]: console log

**Review by PxttytheThxtty — APPROVED** [review]

File: libs/edit-child/feature/src/lib/edit-child.component.html [hunk 3]
```diff
@@ -24,6 +24,15 @@ <h2>Edit Child Details</h2>
                 <p id="surnameError" class="error" style="display: none;">Error in field</p>    
             </div>
 
+
```
Comment (ricecakesreece) [inline libs/edit-child/feature/src/lib/edit-child.component.html:27]: extra spaces before and after

File: libs/edit-child/feature/src/lib/edit-child.component.ts (hunk 1 above)
Comment (ricecakesreece) [reply libs/edit-child/feature/src/lib/edit-child.component.ts:232]: Haha pls reword

**Review by ricecakesreece — APPROVED** [review]

**Review by PxttytheThxtty — APPROVED** [review]

**Pull request merged**